example/install.sh

```diff
@@ -13,8 +13,11 @@ helm upgrade mysql incubator/mysqlha --install ${EXTRA_OPTS} --version 0.4.0 \
 
 helm upgrade wordpress stable/wordpress --install ${EXTRA_OPTS} --version 3.0.2  \
   --values values/wordpress.values.yaml \
+  --set service.annotations."external-dns\.alpha\.kubernetes\.io/hostname"="wordpress.${DOMAIN}" \
```

Comment:
@cliedeman pointed out that this annotation isn't needed, traefik will automatically register it.